core-bundle/contao/templates/backend/be_main.html5

```diff
@@ -37,7 +37,7 @@
   <?php /* If the following hash changes, Turbo will reload the entire page instead of merging contents. */ ?>
   <meta data-turbo-track="reload" content="<?= md5($this->stylesheets."\0".$this->javascripts); ?>">
 </head>
-<body id="top" class="be_main<?php if ($this->isPopup): ?> popup<?php endif; ?>"<?= $this->attributes ?>>
+<body id="top" class="be_main<?php if ($this->isPopup): ?> popup<?php endif; ?>"<?= $this->attributes ?> data-controller="contao--tooltip">
```

Comment:
I think for consistency we should register this controller on `<html>`, just like the other(s).

Reply:
This won't remove the stimulus controller when using Turbo but maybe the already added MutationObserver within this controller will handle new elements - Will try it out.

Reply:
See: https://github.com/contao/contao/pull/7458#issuecomment-2298259378